practical-07/src/components/RegisterForm.vue

```diff
@@ -0,0 +1,152 @@
+<template>
+   <div class="p-4">
+        <div class="card  align-items-center">
+        <Form @submit="handleSubmit" :validation-schema="schema" class="p-6 row col-sm-4">
+            <div class="heading">Sign Up</div>
+               <div class="form-group mb-2">
+                <label for="username">Name :</label>
+                <Field id="username" name="username" type="text" class="form-control" placeholder="Enter name" />
+                <ErrorMessage name="username" class="text-danger" />
+            </div>
+
+            <div class="form-group mb-2">
+                <label for="email">Email address :</label>
+                <Field id="email" name="email" type="email" class="form-control" placeholder="Enter E-mail" />
+                <ErrorMessage name="email" class="text-danger" />
+            </div>
+ 
+           <div class="form-group mb-2">
+                <label for="password">Password :</label>
+                <Field id="password" name="password" type="password" class="form-control" placeholder="Enter Password" />
+                <ErrorMessage name="password" class="text-danger" />
+            </div>
+           
+            <div class="form-group mb-2">
+                <label for="cnfpassword">Confirm Password :</label>
+                <Field id="cnfpassword" name="cnfpassword" type="password" class="form-control" placeholder="********" />
+                <ErrorMessage name="cnfpassword" class="text-danger" />
+            </div>
+
+            <div class="form-group mb-2">
+                <label for="role">Role :</label>
+                <Field id="role" name="role" as="select" class="form-select">
+                    <option value="">please select a role</option>
+                    <option value="admin">Admin</option>
+                    <option value="employee">Employee</option>
+                    <option value="customer">Customer</option>
+                </Field>
+                <ErrorMessage class="text-danger" name="role" />
+            </div>
+
+            <div class="form-group mb-2">
+                <span><label for="gender">Gender : </label></span> 
+                <span><Field name="gender" class="form-check-input" type="radio" value="male"></Field>Male</span>
```

Comment:
Codacy has a fix for the issue: [Replace `······<span><Field·name="gender"·class="form-check-input"·type="radio"·value="male"></Field>Male</span>` with `<span⏎············><Field⏎··············name="gender"⏎··············class="form-check-input"⏎··············type="radio"⏎··············value="male"⏎············></Field⏎············>Male</span`](https://app.codacy.com/gh/mansinakrani/vuejs-practicals/pullRequest?prid=9375994)
```suggestion
          <span
```